spring-integration-core/src/main/java/org/springframework/integration/dsl/ConsumerEndpointSpec.java

```diff
@@ -88,8 +88,8 @@ public S advice(Advice... advice) {
 	}
 
 	/**
-	 * Specify a {@link TransactionInterceptor} {@link Advice} with the
-	 * provided {@code PlatformTransactionManager} and default {@link DefaultTransactionAttribute}
+	 * Specify a {@link TransactionInterceptor} {@link Advice} with the provided
+	 * {@code PlatformTransactionManager} and default {@link DefaultTransactionAttribute}
 	 * for the {@code pollingTask}.
```

Comment:
I think this is wrong. 
The main premise of the TX advice here to wrap `handleMessage()` operation.
There is nothing about `pollingTask`.

Yes, that is my fault, but maybe you wouldn't mind to fix it here :smile: 

Thanks